### PR TITLE
feat: test results summary

### PR DESCRIPTION
**Pull request by eric-hc**
- show [result summary](https://k6.io/docs/results-visualization/end-of-test-summary/) for each test result 
- minor table enhancements
- use `execa`
- implement delete a test result 
- results page, show summary

### CODE REVIEW

File: src/components/projects/file.jsx [hunk 1]
```diff
@@ -94,23 +91,6 @@ export const File = props => {
       headers: {
         'Content-Type': 'application/json'
       }
-    }).then(res => {
```
Comment (swand0g) [inline src/components/projects/file.jsx:97]: so long 🦀

File: src/components/projects/file.jsx [hunk 2]
```diff
@@ -232,8 +192,8 @@ export const Results = props => {
                   }}
                   variant="outline"
                 >
-                  <Button>Previous</Button>
-                  <Button>Next</Button>
+                  <Button disabled>Previous</Button>
```
Comment (swand0g) [inline src/components/projects/file.jsx:195]: we should create issue for this

File: src/components/sidebar/sidebar.jsx [hunk 3]
```diff
@@ -58,12 +54,24 @@ export default function Sidebar({ user }) {
...
+              </Stack>
+            </NavSection>
+            <NavSection title="Resources">
+              <Stack pb="6">
+                <SidebarLink href="https://k6.io/docs" icon={<Mountain size={15} />}>
+                  k6 docs
+                </SidebarLink>
+                <SidebarLink href="https://jslib.k6.io" icon={<Wrench size={15} />}>
+                  jslib
+                </SidebarLink>
```
Comment (swand0g) [inline src/components/sidebar/sidebar.jsx:73]: Niceeee

File: src/pages/api/results/[id].ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      const data = await result.findUnique({
+        where: {
+          id: Number(id)
+        },
+        include: {
+          user: true
+        }
+      })
+      console.log(`👹 ~ file: [id].ts ~ line 17 ~ handler ~ data`, data)
+      res.status(200).json({
```
Comment (swand0g) [inline src/pages/api/results/[id].ts:20]:
```suggestion
      })
      res.status(200).json({
```

File: src/pages/api/tests/index.ts [hunk 5]
```diff
@@ -2,47 +2,62 @@ import { NextApiRequest } from 'next'
...
-        path: `${repo}/${path.join('/')}`
-      },
-      include: {
-        results: {
-          include: {
-            user: true
+  switch (method) {
+    case 'POST':
+      const { repoId, path, repo } = JSON.parse(req.body)
+      // is the test exist
```
Comment (swand0g) [inline src/pages/api/tests/index.ts:11]: WHAT 🤨

File: src/pages/api/results/[id].ts (hunk 4 above)
Comment (eric-hc) [reply src/pages/api/results/[id].ts:20]: dude quit beansing me

**Review by swand0g — APPROVED** [review]

File: src/components/projects/file.jsx (hunk 2 above)
Comment (tiffkwin) [reply src/components/projects/file.jsx:195]: do it then

**Pull request merged**